subhub/tests/bdd/steps/test_utils.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+import json
+
+
+def _generate_pprint_json(value):
+    return json.dumps(value, sort_keys=True, indent=4)
+
+
+def generate_assert_failure_message(expected_value, actual_value):
+    return "\033[91m\n\nExpected:\n{0}\n\nReceived:\n{1}\n\n\033[0m".format(
```

Comment:
shorten _genereate_pprint_json function name and use inline as f-string instead of format with positional {0}